hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/client/HdfsClientConfigKeys.java

```diff
@@ -273,6 +273,14 @@ public interface HdfsClientConfigKeys {
   String DFS_LEASE_HARDLIMIT_KEY = "dfs.namenode.lease-hard-limit-sec";
   long DFS_LEASE_HARDLIMIT_DEFAULT = 20 * 60;
 
+  String DFS_CLIENT_FSCK_CONNECT_TIMEOUT_MS =
+      "dfs.client.fsck.connect.timeout.ms";
+  int DFS_CLIENT_FSCK_CONNECT_TIMEOUT_MS_DEFAULT = 60 * 1000;
+
+  String DFS_CLIENT_FSCK_READ_TIMEOUT_MS =
+      "dfs.client.fsck.read.timeout.ms";
+  int DFS_CLIENT_FSCK_READ_TIMEOUT_MS_DEFAULT = 60 * 1000;
```

Comment:
do you want to restrict the config to `ms`? Can keep only ``dfs.client.fsck.connect.timeout`` similar to ``dfs.webhdfs.socket.connect-timeout`` and allow timeunits?

Reply:
This suggestion makes sense. I have changed it to use the same technique as with webhdfs.